salt/transactional-update/10-update-rebootmgr-options.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
```

Comment:
To override the whole `ExecStart` instead of appending to it, systemd insists on a blank one.